docs/raredisease_map.md

```diff
@@ -0,0 +1,81 @@
+| chromograph_cov, tcov                             | False       | chromograph, tcov                       | scout        |
+| smncopynumbercaller, tsv                          | False       | smn-calling                             | scout        |
+| expansionhunter, variant_catalog                  | False       | expansionhunter, variant-catalog        | scout        |
+| expansionhunter, case_sv_str                      | False       | vcf-str                                 | scout        |
+| expansionhunter, case_sv_str_index                | False       | vcf-str_index                           | scout        |
+| expansionhunter, str_variants                     | False       | expansionhunter, vcf-str                | scout        |
+| expansionhunter, str_variants_index               | False       | expansionhunter, vcf-str-index          | scout        |
+| expansionhunter, str_alignment                    | False       | expansionhunter, bam                    | scout        |
+| expansionhunter, str_alignment_index              | False       | expansionhunter, bam-index              | scout        |
+| vcf2cytosure, vcf2cytosure                        | False       | vcf2cytosure                            | scout        |
+| gens_generatedata, baf                            | False       | gens, fracsnp, bed                      | scout        |
+| gens_generatedata, baf_index                      | False       | gens, fracsnp, bed-index                | scout        |
+| gens_generatedata, cov                            | False       | gens, coverage, bed                     | scout        |
+| gens_generatedata, cov_index                      | False       | gens, coverage, bed-index               | scout        |
+| call_mobile_elements, call_mobile_elements        | False       | gens, coverage, bed-index               | scout        |
+| call_mobile_elements, call_mobile_elements_index  | False       | gens, coverage, bed-index               | scout        |
+| annotate_mobile_elements, clinical                | False       | mobile-elements, clinical, vcf          | scout        |
+| annotate_mobile_elements, clinical_index          | False       | mobile-elements, clinical, vcf_index    | scout        |
+| annotate_mobile_elements, research                | False       | mobile-elements, research, vcf          | scout        |
+| annotate_mobile_elements, research_index          | False       | mobile-elements, research, vcf_index    | scout        |
+| call_snv, call_snv                                | True        | vcf_snv                                 | genotype     |
+| call_snv, call_snv_index                          | False       | vcf_snv_index                           | scout        |
+| call_snv, call_snv_mt                             | False       | vcf_snv, mitochondria                   | scout        |
```

Comment:
remove `_` in favour of `-` for all the cases

and separate vcf or vcf-index from the type? `snv` `str` etc?

Reply:
Same here:https://github.com/Clinical-Genomics/hermes/pull/109#discussion_r1598319025